src/renderer/component/ToggleButtons/LayoutViewSelector.js

```diff
@@ -26,6 +26,20 @@ import { ButtonConfig } from "../Button";
 import { IconEditModeStandardViewSm, IconEditModeSingleViewSm } from "../../component/Icon";
 
 const Style = Styled.div`
+// WHY: We want to hide the selector when we cannot use it (e.g. when color editor is active)
+${({ isHidden }) =>
+  isHidden &&
+  `
+  display: none;
+`}
```

Comment:
It might actually be better to not render the component at all -> Remove style and use conditional rendering